src/rgw/rgw_tracer.cc

```diff
@@ -0,0 +1,133 @@
+
+#include "rgw_tracer.h"
+
+using jConfig = jaegertracing::Config;
+using jLogger = jaegertracing::logging::Logger;
+
+namespace RGW_Tracing{
+
+    /*
+    //  Tracer
+    //
+    */
+    const std::shared_ptr<opentracing::Tracer> Tracer::noop_tracer = opentracing::MakeNoopTracer();
+    TracerConfiguration Tracer::tracer_conf;
+    
+    Tracer::Tracer(jConfig& conf):open_tracer(jaegertracing::Tracer::make(conf)){}
+    Tracer::Tracer(
+    std::string& service_name, 
+    const jConfig& conf, 
+    const std::shared_ptr<jLogger>& logger)
```

Comment:
for methods inlined in a header, most of ceph rgw seems to use:
void method(args) {
/* stuff */
}